src/Symfony/Bundle/SecurityBundle/Resources/config/schema/security-1.0.xsd

```diff
@@ -321,11 +300,11 @@
     <xsd:complexType name="oidc_token_handler">
         <xsd:sequence>
             <xsd:choice minOccurs="0" maxOccurs="1">
-                <xsd:element name="oidc-user-info" type="oidc_user_info"></xsd:element>
-                <xsd:element name="oidc" type="oidc"></xsd:element>
+                <xsd:element name="oidc-user-info" type="oidc_user_info" />
+                <xsd:element name="oidc" type="oidc" />
             </xsd:choice>
         </xsd:sequence>
-        <xsd:attribute name="oidc-user-info" type="xsd:anyURI"></xsd:attribute>
+        <xsd:attribute name="oidc-user-info" type="xsd:anyURI" />
```

Comment:
This should probably also be changed to `xsd:string"`, to allow using DIC parameters in the config (and maybe even env placeholders in those parameters, if the processing of the config supports them for this setting).
Requiring a URI at the XSD level is too restrictive.